src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,131 @@
+package com.epam.tc.hw4.ex1;
+
+import com.epam.tc.hw4.AbstractChromeTest;
+import io.qameta.allure.Description;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Step;
+import io.qameta.allure.Story;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.Test;
+
+public class Exercise1Test extends AbstractChromeTest {
+
+    SoftAssertions softly = new SoftAssertions();
+
+    private static final int COUNT_OF_IMAGES_ON_MAIN_PAGE = 4;
+    public static final List<String> LEFT_MENU_ITEMS = List
+            .of("Home", "Contact form", "Service", "Metals & Colors", "Elements packs");
+    public static final List<String> LIST_ITEMS_HEADER = List
+            .of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
+    private static final List<String> TEXT_UNDER_IMAGES = List
+            .of("To include good practices\nand ideas from successful\nEPAM project",
+                    "To be flexible and\ncustomizable",
+                    "To be multiplatform",
+                    "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…");
+
+    @Test(groups = {"HW4"})
+    @Description(value = "Test case for Exercise #1")
+    @Feature("Feature One")
+    @Story("Index Page")
+    public void exercise1() {
+
+        assertBrowserTitle();
+        performLogin();
+        assertUserLoggined();
+        assertItemsInHeader();
+        assertImagesOnIndexPage();
+        assertTextUnderIcons();
+        assertIframeWithButton();
+        switchToFrame();
+        switchToWindow();
+        assertItemsInLeftSection();
+        softly.assertAll();
+    }
+
```

Comment:
It's not a good idea to have your test steps in test class. Please, create a new package for your test steps. There you could create base class, e.g., BaseStep or AbstractStep, that would be extended by classes with Action steps and Assertion steps.